modules/appeals_api/spec/workers/higher_level_review_pdf_submit_job_spec.rb

```diff
@@ -34,6 +34,7 @@
     expect(capture_body).to have_key('metadata')
     expect(capture_body).to have_key('document')
     metadata = JSON.parse(capture_body['metadata'])
+    expect(metadata['veteranFirstName']).to eq('Jane')
```

Comment:
While this test is definitely worthwhile to have, I think that since `transliterate_name` does so much (translitreating, removing chars, stripping, and truncating), we should specifically test it directly and not just its result to the metadata.

Reply:
I see. Yes yes yes